integration_tests/src/main/python/hybrid_parquet_test.py

```diff
@@ -227,3 +227,19 @@ def udf_fallback(s):
     assert_gpu_and_cpu_are_equal_collect(
         lambda spark: spark.read.parquet(data_path).filter("ascii(a) >= 50 and udf_fallback(a) = 'udf_100'"),
         conf=filter_split_conf)
+
+@allow_non_gpu(*non_utc_allow)
+def test_hybrid_parquet_filter_pushdown_timestamp(spark_tmp_path):
```

Comment:
Does this need to be marked as a hybrid test?

Reply:
nice catch, thanks